WordPress/src/main/java/org/wordpress/android/util/config/ContactInfoBlockFeatureConfig.kt

```diff
@@ -1,9 +0,0 @@
-package org.wordpress.android.util.config
-
-import org.wordpress.android.BuildConfig
-import org.wordpress.android.annotation.FeatureInDevelopment
-import javax.inject.Inject
-
-@FeatureInDevelopment
```

Comment:
I see instances of features marked `@FeatureInDevelopment` become `@Feature` (e.g. [here](https://github.com/wordpress-mobile/WordPress-Android/commit/253845bfd06ee56d14975e37f5d1b611cc835695#diff-0dbe0eaa2da97e217e9514c561ae063c3fe828ab3679205a17df36bf606365c7)), so I'm not sure which approach to take.

Reply:
I do see some feature flags deleted though (e.g. [here](https://github.com/wordpress-mobile/WordPress-Android/commit/3e9d7b20a189609de9c5de0e1b646f6ea8b925c8#diff-0dbe0eaa2da97e217e9514c561ae063c3fe828ab3679205a17df36bf606365c7)), which seems similar to what I'm doing here.

Reply:
> I see instances of features annotated with `@FeatureInDevelopment` become `@Feature` (e.g. [here](https://github.com/wordpress-mobile/WordPress-Android/commit/253845bfd06ee56d14975e37f5d1b611cc835695#diff-0dbe0eaa2da97e217e9514c561ae063c3fe828ab3679205a17df36bf606365c7)), so I'm not sure which approach to take.

Aha, looking at [this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/14734), `@Feature` might be used when remote feature flagging is needed, which I don't this is a need here.
I'm leaving this comment thread unresolved in case there's a better way of doing this.